crypto/address.go

```diff
@@ -11,14 +11,18 @@ import (
 	"github.com/zarbchain/zarb-go/errors"
 )
 
-// Address format
-// `zc` + type + data
-// type is 1 for schnorr signatures
+// Address format:
+// `zc` + type + data + checksum
+// type is 1 for BLS signatures
 
 const (
-	addressSize           = 20
+	AddressTypeBLS byte = 1
+)
```

Comment:
maybe use `iota` instead of 1?!

Reply:
Good idea. Right now we have only one type of address.